dev/README.md

```diff
@@ -427,3 +427,69 @@ backwards compatible
 
 See [django-migration-linter checklist](https://github.com/3YOURMIND/django-migration-linter/blob/main/docs/incompatibilities.md)
 for the common mistakes and best practices
+
+### Removing a nullable field from a model
+
+> This only works for nullable fields (fields with `null=True` in the field definition).
+>
+> DO NOT USE THIS APPROACH FOR NON-NULLABLE FIELDS, IT WILL BREAK THINGS!
```

Comment:
what if the field has the default value on the database level? We had a check for that in migration linter

Reply:
If a field is non-nullable and has a default value on the DB level, it should be fine to use the same approach. This section is only about nullable fields, so I don't want to include this info here. I'll change `IT WILL BREAK THINGS` to `IT CAN BREAK THINGS` so it's more precise.

Reply:
Non-nullable fields are a bit trickier, my intention is to scope this PR purely to nullable fields for now.